subsys/partition_manager/CMakeLists.txt

```diff
@@ -36,13 +30,11 @@ if((EXISTS ${APPLICATION_SOURCE_DIR}/pm.yml) AND IMAGE)
   if(NOT "${ret}" STREQUAL "0")
     message(FATAL_ERROR "command failed with return code: ${ret}")
   endif()
-  string(LENGTH ${IMAGE} len)
-  MATH(EXPR len "${len}-1")
-  string(SUBSTRING ${IMAGE} 0 ${len} image_name)
+
   set_property(
     GLOBAL APPEND PROPERTY
-    PARTITION_MANAGER_CONFIG_FILES
-    "${image_name}|${PROJECT_BINARY_DIR}/include/generated/pm.yml|${PROJECT_BINARY_DIR}|${PROJECT_BINARY_DIR}/include/generated"
```

Comment:
I'm assuming this is a breaking change with Zephyr, if so the manifest update must be in the same commit to retain bisectability.

Reply:
I updated the commit history to be bisectable (one west.yml change per commit + merge doc into hex file commit)